mobile/lib/features/trade/application/order_service.dart

```diff
@@ -7,6 +9,10 @@ import 'package:get_10101/features/trade/domain/direction.dart';
 import 'package:get_10101/features/trade/domain/leverage.dart';
 import 'package:get_10101/features/trade/domain/api_response.dart';
 
+import 'package:get_10101/features/trade/domain/order.dart';
+
+import '../order_change_notifier.dart';
```

Comment:
🔧 can we stick to using the package import instead of the relative.. that makes it easier to move files.

Reply:
It's annoying that the IDE keeps importing them like this - I tried to find a way to change the default but could not find a setting for it - did you manage to configure package imports as default?